zqd/space/space.go

```diff
@@ -116,10 +147,28 @@ type config struct {
 	PacketPath string `json:"packet_path"`
 }
 
+type info struct {
+	MinTime nano.Ts `json:"min_time"`
+	MaxTime nano.Ts `json:"max_time"`
+}
+
+func (s Space) SetTimes(minTs, maxTs nano.Ts) error {
+	cur, err := loadInfo(s.path)
```

Comment:
This should be in the data path, not the space path.

Reply:
Moved.